test/routes/previous-application-number.route.test.js

```diff
@@ -66,6 +67,11 @@ describe('/previous-application-number route', () => {
       )
     })
 
+    it('should have callOutText', () => {
+      const element = document.querySelector(`#${elementIds.callOutText}`)
+      expect(element).toBeTruthy()
```

Comment:
Also need to check the content of the element:

expect(TestHelper.getTextContent(element)).toEqual('...')